packages/react-select/src/Select.js

```diff
@@ -1261,16 +1177,11 @@ export default class Select extends Component<Props, State> {
     }
   };
   onInputFocus = (event: SyntheticFocusEvent<HTMLInputElement>) => {
-    const { isSearchable, isMulti } = this.props;
     if (this.props.onFocus) {
       this.props.onFocus(event);
     }
-    this.setState({ inputIsHiddenAfterUpdate: false });
-    this.announceAriaLiveContext({
-      event: 'input',
-      context: { isSearchable, isMulti },
-    });
     this.setState({
+      inputIsHiddenAfterUpdate: false,
       isFocused: true,
     });
```

Comment:
I added this after line 1186 to test screenreader with default value, and it seems to be working quite well:

```ts
this.ariaOnChange(this.props.value, {
  action: 'select-option',
});
```

For your consideration